test/with_dummyserver/test_https.py

```diff
@@ -272,7 +276,7 @@ def test_context_combines_with_ca_certs(self):
                 assert not warn.called, warn.call_args_list
 
     @notSecureTransport()  # SecureTransport does not support cert directories
-    def test_ca_dir_verified(self, tmpdir):
+    def test_ca_dir_verified(self, tmpdir: Path) -> None:
```

Comment:
This should be `tmpdir: py.path.local` (though might as well replace it with `tmp_path: Path` as mentioned above)